e3nn/nn/_activation.py

```diff
@@ -95,7 +95,8 @@ def forward(self, features, dim: int = -1):
         # - PROFILER - with torch.autograd.profiler.record_function(repr(self)):
         output = []
         index = 0
-        for mul, (l, _), act in self.paths:
+        # for mul, (l, _), act in self.paths:
```

Comment:
Thanks ! Can you delete the commented out line instead of commenting it for clarity ?